pallets/eth-bridge/src/eth.rs

```diff
@@ -14,10 +14,14 @@ use sp_runtime::DispatchError;
 use sp_std::vec;
 
 pub fn sign_msg_hash<T: Config<I>, I: 'static>(
+    author: &Author<T>,
     msg_hash: &H256,
 ) -> Result<ecdsa::Signature, DispatchError> {
-    let msg_hash_string = hex::encode(msg_hash);
-    let confirmation = AVN::<T>::request_ecdsa_signature_from_external_service(&msg_hash_string)?;
+    let msg_data = msg_hash.as_ref().to_vec();
+    let hex_data = hex::encode(&msg_data).into_bytes();
```

Comment:
Unless I am mistaken, `msg_data` and `hed_data` are both bytes for the same thing. Wouldn't it be better to sign a hex encoded proof and request signature on the original msg_data? In other words, 
- the proof that we sign is the hex encoded data
- the data we collect signatures for stays as the original bytes